R/rprofile.R

```diff
@@ -40,7 +40,7 @@ use_devtools <- function() {
 use_rprofile_package <- function(package) {
   check_installed(package)
   ui_bullets(c(
-    "_" = "Include this code in {.path .Rprofile} to make {.pkg {package}}
+    "_" = "Include this code in {.href [.Rprofile](file://~/.Rprofile)} to make {.pkg {package}}
```

Comment:
This hard-wired `.Rprofile` path makes me nervous and I think it's inconsistent with what we do elsewhere in usethis re: this path.

Reply:
The only problem with the current approach is that it either creates a clickable output. However, clicking on it will do nothing..

 (that feels confusing to me.) So if you don't think this is safe enough, maybe using {.val} would be better?

Reply:
I'm worried about the path to `.Rprofile`. It's not necessarily/always `~/.Rprofile`. One has to honor the `R_PROFILE_USER` env var, for example. And you have to worry about whether the whole stack has the same idea of "home directory" on Windows.

https://github.com/r-lib/usethis/blob/682f4bd92879367bebc1cab9ac2c8077733048a4/R/edit.R#L113

Reply:
I have decided to revert as you responded :)